### PR TITLE
bind SPC gA to magit-cherry-pick-popup

### PR DESCRIPTION
**Pull request by robbyoconnor**
Bind `magit-cherry-pick-popup` to `SPC gA`

### CODE REVIEW

**Comment by justbur** [comment]
`A` is cherry pick in magit if you want to be consistent.

**Comment by robbyoconnor** [comment]
@justbur fixing :)

**Comment by robbyoconnor** [comment]
@justbur fixed -- thank you so much :heart:

**Comment by justbur** [comment]
:+1:

File: layers/+source-control/git/packages.el [hunk 1]
```diff
@@ -96,6 +96,7 @@
 (defun git/init-magit ()
   (use-package magit
     :commands (magit-blame-mode
+               magit-cherry-pick-popuF
```
Comment (StreakyCobra) [inline layers/+source-control/git/packages.el]: What is a `popuF` :joy: ?

**Comment by StreakyCobra** [comment]
:+1: Indeed consistency is the most important, so `gA` is the way to go!

File: layers/+source-control/git/packages.el (hunk 1 above)
Comment (robbyoconnor) [inline layers/+source-control/git/packages.el]: It's called a typo :P Fixed.

**Comment by TheBB** [comment]
Thanks! Cherry-picked in develop. You can safely delete your branch.

**Pull request closed without merging**